xml/System.Text.Json.Serialization/JsonConverterAttribute.xml

```diff
@@ -77,7 +89,7 @@
         <ReturnType>System.Type</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The type of the converter to create, or <see langword="null" /> if <see cref="M:System.Text.Json.Serialization.JsonConverterAttribute.CreateConverter(System.Type)" /> should be used to obtain the converter.</summary>
```

Comment:
I really do not understand this paragraph.
I think it should say something like this:
```suggestion
        <summary>Gets the type of the <see cref="T:System.Text.Json.Serialization.JsonConverterAttribute"/>, or <see langword="null" /> if it was created without a type.</summary>
```